src/vivarium_gates_bep/components/lbwsg.py

```diff
@@ -60,13 +61,8 @@ def on_initialize_simulants(self, pop_data):
 
 
 def read_bw_bin_data(builder, key):
-    path = builder.configuration.input_data.artifact_path
-    key = key.replace(".", "/")
-    with pd.HDFStore(path, mode='r') as store:
-        data = store.get(f'/{key}')
-        data = data.rename(columns={'bw_lower_bound': 'birth_weight_start', 'bw_upper_bound': 'birth_weight_end'})
-        data['Value'] = data.index / len(data)
-    return data
+    art = Artifact(builder.configuration.input_data.artifact_path)
+    return art.load(key)
```

Comment:
What is preventing you from doing `builder.data.load(key)`?

Reply:
Whatever it is, you should either fix it on the build side or document the reason you can't use the standard tools here.

Reply:
art.load('birth_weight.bins') produces a dataframe with birth_weight_start, birth_weight_end, value
and that works with this call:
builder.lookup.build_table(bw_data, parameter_columns=['birth_weight'])

builder.data.load('birth_weight.bins') strips the birth_weight* columns leaving just index, value
and I didn't find a variation of builder.lookup.build_table() that worked with that structure